lib/datasource/hex/index.ts

```diff
@@ -7,7 +7,10 @@ export const id = 'hex';
 interface HexRelease {
   html_url: string;
   meta?: { links?: Record<string, string> };
-  releases?: { version: string }[];
+  releases?: {
+    version: string;
+    inserted_at;
```

Comment:
Is it really mandatory/always present?

Reply:
Looks like it is, but added condition anyway